Bookcrossing.Application/Commands/Broker/NotificationCommand.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Bookcrossing.Application.Handler;
+using Bookcrossing.Application.Logger;
+using Bookcrossing.Contracts.DataTransferObjects.Broker;
+using MassTransit;
+using MediatR;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Bookcrossing.Application.Commands.Broker
+{
+    public class NotificationCommand : IRequest
```

Comment:
name it using verbs, also related to SubscribtionCommand and UnsubscribtionCommand